csm/conf/configure.py

```diff
@@ -372,7 +375,7 @@ def _configure_csm_ldap_schema(self):
                     self._create_csm_ldap_user(each_server_url)
                     # Setup necessary permissions
                     Log.info(f"Setup necessary permissions to {ldap_user} on server: {each_server_url}")
-                    self._setup_ldap_permissions(each_server_url, base_dn, ldap_user)
+                    self._setup_ldap_permissions(each_server_url, base_dn, ldap_user,csm_schema_version)
```

Comment:
you may not need this param -> 'csm_schema_version' in  _setup_ldap_permissions(). please check.

Reply:
Done

Reply:
You can resolve this.